layouts/shortcodes/docs/languages/exporters/intro.md

```diff
@@ -34,7 +34,7 @@ Specification][OTLP].
 [Jaeger]: /blog/2022/jaeger-native-otlp/
 [OTLP]: /docs/specs/otlp/
 [Prometheus]:
-  https://prometheus.io/docs/prometheus/latest/feature_flags/#otlp-receiver
+  https://prometheus.io/docs/prometheus/2.55/feature_flags/#otlp-receiver
```

Comment:
This fixes the link, but the fact that an OTLP-receiver entry no longer exists in the latest version of the docs is probably an indication that this prose needs to be updated (and/or the link to point to something else in the latest docs)?

Reply:
@open-telemetry/collector-approvers ☝️